includes/render.php

```diff
@@ -61,7 +61,7 @@ class GEM_Form_Fields {
 
 	private static $cycle = 0;
 
-	public function dispatch_field( $field, $cycle = 1 ) {
+	public static function dispatch_field( $field, $cycle = 1 ) {
```

Comment:
They are called statically, for example:
GEM_Form_Fields::dispatch_field( $field, self::$loops )